aldatemplates/models/pms_property.py

```diff
@@ -10,6 +10,8 @@ class PmsProperty(models.Model):
         string="Text Image City",
     )
     text_image2 = fields.Html(string="Text Image Property")
-    web_city_information_url = fields.Char(string="Web City Info URL")
+    web_city_information_url = fields.Html(string="Web City Info URL")
```

Comment:
Being a URL field, I don't see why it should be of type HTML; this will make the link not work. Moreover, even if we did it that way, we would need to add a small migration script to transfer the information from one field to another without losing any data.
I think I would leave it as a Char field, right?